packages/digit_data_model/lib/data/repositories/local/base/service_attributes_base.dart

```diff
@@ -1,12 +1,12 @@
-import 'package:digit_data_model/data_model.dart';
-import 'package:drift/drift.dart';
-
-abstract class ServiceAttributesLocalBaseRepository
-    extends LocalRepository<ServiceAttributesModel, ServiceAttributesSearchModel> {
-  const ServiceAttributesLocalBaseRepository(super.sql, super.opLogManager);
-
-  @override
-  DataModelType get type => DataModelType.serviceAttributes;
-
-  TableInfo get table => sql.serviceAttributes;
-}
+// import 'package:digit_data_model/data_model.dart';
```

Comment:
what is the reason for removal of this code

Reply:
> what is the reason for removal of this code

I added it back.